### ISSUE TITLE
5장

### ISSUE THREAD

**Issue by MoSonLee** [MoSonLee/ArtificialIntelligence#4]

**Comment by MoSonLee** [comment]
### 5장 용어 정리
   - Supervised learning(지도 학습): 훈련 데이터로부터 하나의 함수를 유추해내기 위한 기계 학습의 방법
   - training set: 훈련 데이터( x, y의 집합)
   - predictors, target: (x, y)에서 x를 predictor, y를 target이라 한다.
   - model: 함수를 model이라고 부름
   - Regression: 통계적 수치를 나타내는 분석 기술( 연속적일 수도 있고 아닐 수도 있음)
      - Simple linear regression
         
![image](https://user-images.githubusercontent.com/77050826/145704547-6259ae43-2b5c-4b06-95dd-5abd07ca18a1.png)
         - x와 y의 값이 하나임
         
      - Multiple linear regression
       
![image](https://user-images.githubusercontent.com/77050826/145704559-89a7509a-db4d-456f-ae7e-a36f600498c1.png)
      - x0, x1과 같이 x와 y의 값이 여러개임

   - Overfitting: 모델이 데이터를 너무 잘 구별 했을 때 결과가 안 좋을 경우 overfitting 됐다고 말한다.
      - trainning set에는 너무 잘 맞지만 test set에서는 오류가 났을 때
      - 어떻게 overfitting을 피할 수 있을까?
         - **test set**이 필요하다. -> training set으로만 학습시키는 것은 좋지 않다. 
         - **LOOCV method(Leave-One-Out Cross Validation)** : 교차 검증을 한다.
         
   - Simple linear regression
      - 평균제곱오차가 minimize 되게 구한다.(B0와 B1)
      - 그 구한값과 실제 값과의 차이가 **loss function** (예상 y와 실제 y의 차이를 구하는 것)

   
![image](https://user-images.githubusercontent.com/77050826/145705409-8a9039d1-d135-4b77-a1c3-46b97bf2f9c3.png)

   - Gradient descent(미분 감소): 경사하강법(절댓값이 점점 작은 값으로 이동시키기 위해 반복)(거의 변화가 없으면 그만둠)
      - cost function을 minimize하는 것

   - Decision Tree
      - x로부터 y에게 함수를 공급하는 tree(모든 함수가 분명할 때 x로부터 y의 함수를 제공)
